tutorials/equations.ipynb

```diff
@@ -300,7 +300,7 @@
       "        self.reset_state_from_file()\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    fluid_flux_data
</pre><p>See previous comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1439/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
Changed to `print(fluid_flux_data)`